File: E_Kotlin/Projects/JuegoSnake/app/src/main/java/com/sebss/juegosnake/juego/Civil.java
package com.sebss.juegosnake.juego;

public class Civil extends Entidad{
    public static final int YAYA = 0;
    public static final int NINO = 1;
    public static final int CANI = 2;

    public int tipo;

    public Civil(int x,int y,int tipo){
        super(x,y);
        this.tipo=tipo;
    }
}

File: E_Kotlin/Projects/3DGraphicsMotor/app/src/main/java/com/sebss/a3dgraphicsmotor/juego/Civil.kt
package com.sebss.a3dgraphicsmotor.juego

class Civil(x: Int, y: Int, @JvmField var tipo: Int) : Entity(x, y) {
    companion object {
        const val YAYA = 0
        const val NINO = 1
        const val CANI = 2
    }
}